src/components/onboarding/need-wallet/index.js

```diff
@@ -0,0 +1,78 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import React from 'react'
+
+import ROUTE from 'constants/route'
+import WalletButton from 'components/common/wallet-button'
+
+import vectorImg from './Vector.png'
+import style from './index.style.js'
+
+export default class HomeScene extends React.Component {
+  render() {
+    return (
+      <div css={style.base}>
+        {this.renderBackButton()}
+        {this.renderTitle()}
+        {this.renderRecommendedWallets()}
+        <div css={style.buttonCollection}>
+          {this.renderDownloadStatus()}
+          {this.renderDownloadCoinbase()}
+          {this.renderDownloadTrust()}
+        </div>
+      </div>
+    )
+  }
+
+  renderBackButton() {
+    return (
+      <div css={style.button__location}>
+        <button css={style.continue__button}>
+          <img css={style.vector} src={vectorImg} alt="" />
+        </button>
+      </div>
```

Comment:
Back button should be moved out to a reusable component

Reply:
moved to components/common/back-button